Tensile/KernelWriterSource.py

```diff
@@ -111,22 +85,13 @@ def __init__( self, kernelMinNaming, kernelSerialNaming ):
   # Open String
   ##############################################################################
   def openString(self, kernel):
-    kStr = ""
-    if self.language == "OCL":
-      kernelName = self.getKernelName(kernel)
-      kStr += "\n"
-      kStr += "std::string %s_src_%u = \"" % (kernelName, self.stringIdx)
-    return kStr
+    return ""
```

Comment:
openString and closeString actually does nothing for both Asm and Source.
Maybe we can remove the function itself.